src/main/target/REVONANO/target.h

```diff
@@ -36,15 +36,19 @@
 #define GYRO_1_SPI_INSTANCE     SPI2
 
 #define USE_ACC
-#define USE_ACC_SPI_MPU6500
+#define USE_ACC_SPI_MPU9250
 
 #define USE_GYRO
-#define USE_GYRO_SPI_MPU6500
+#define USE_GYRO_SPI_MPU9250
 #define GYRO_1_ALIGN            CW270_DEG
 
 #define USE_BARO
 #define USE_BARO_MS5611
 
+#define USE_MAG
```

Comment:
Seems to be weird to have legacy target definition changes mangled in here, but since they are legacy and not used anyway I guess it does not matter.